pypeit/reduce.py

```diff
@@ -749,60 +938,69 @@ def find_objects_pypeline(self, image, std=False, ir_redux=False, std_trace = No
+            std_trace=std_trace,
+            specobj_dict=specobj_dict,sig_thresh=self.par['scienceimage']['findobj']['sig_thresh'],
+            show_peaks=show_peaks, show_fits=show_fits,
+            trim_edg=self.par['scienceimage']['findobj']['find_trim_edge'],
+            cont_fit=self.par['scienceimage']['findobj']['find_cont_fit'],
+            npoly_cont=self.par['scienceimage']['findobj']['find_npoly_cont'],
+            fwhm=self.par['scienceimage']['findobj']['find_fwhm'],
+            maxdev=self.par['scienceimage']['findobj']['find_maxdev'],
+            max_snr=self.par['scienceimage']['findobj']['ech_find_max_snr'],
+            min_snr=self.par['scienceimage']['findobj']['ech_find_min_snr'],
+            nabove_min_snr=self.par['scienceimage']['findobj']['ech_find_nabove_min_snr'],
             show_trace=show_trace, debug=debug)
 
         # Steps
         self.steps.append(inspect.stack()[0][3])
         if show:
-            self.show('image', image=image*(self.sciImg.mask == 0), chname = 'ech_objfind',sobjs=sobjs_ech, slits=False)
+            self.show('image', image=image*(self.sciImg.mask == 0), chname='ech_objfind',sobjs=sobjs_ech, slits=False)
 
         return sobjs_ech, len(sobjs_ech), skymask
 
 
     # JFH TODO Should we reduce the number of iterations for standards or near-IR redux where the noise model is not
     # being updated?
     def local_skysub_extract(self, waveimg, global_sky, sobjs,
-                             spat_pix=None, model_noise=True, min_snr=2.0, std = False, fit_fwhm=False,
-                             maskslits=None, show_profile=False, show_resids=False, show_fwhm=False, show=False):
+                             spat_pix=None, model_noise=True, min_snr=2.0, fit_fwhm=False,
+                             show_profile=False, show_resids=False, show_fwhm=False, show=False):
         """
         Perform local sky subtraction, profile fitting, and optimal extraction slit by slit
 
         Wrapper to skysub.local_skysub_extract
 
-        Parameters
-        ----------
-        sobjs: object
-           Specobjs object containing Specobj objects containing information about objects found.
-        waveimg: ndarray, shape (nspec, nspat)
-           Wavelength map
-
-        Optional Parameters
-        -------------------
+        Args:
+            waveimg (np.ndarray):
+            global_sky (np.ndarray):
+            sobjs (SpecObjs):
+            spat_pix (np.ndarray, optional):
+            model_noise (bool, optional):
+            show_resids (bool, optional):
+            show_profile (bool, optional):
+            show (bool, optional):
 
         Returns:
-            global_sky: (numpy.ndarray) image of the the global sky model
-        """
+            tuple: skymodel (np.ndarray), objmodel (np.ndarray), ivarmodel (np.ndarray), outmask (np.ndarray), sobjs
 
+        """
         self.waveimg = waveimg
         self.global_sky = global_sky
 
```

Comment:
Is this redundant with the get_platescale methods you just added?

Reply:
no.